src/yt_utils.py

```diff
@@ -26,19 +26,25 @@ def get_best(formats):
 
 
 def get_best_audio(formats):
-    filtered = list(filter_audio(formats))
+    filtered = list(filter_video(formats))
+    if not filtered:
+        filtered = list(filter_audio_video(formats))
```

Comment:
`get_best_audio` should return audio-only. I think there is nothing wrong with the previous implementation.

Reply:
honestly, didn't checked where `get_best_audio` used and why this method needed but previous implementation will at least need check for empty filtered list before `min` or `max`

Reply:
I agree with you. Maybe you should revert the line 29, but keep the changes that you had made on line 30 and 31.

Reply:
ahhh line 29 change is just typo, sorry

Reply:
Thanks